packages/server/utils/setUserTierForUserIds.ts

```diff
@@ -28,14 +28,33 @@ const setUserTierForUserIds = async (userIds: string[]) => {
       )
     }))
     .run()) as {id: string; tier: TierEnum}[]
-  const newUserTiers = userIds.map((userId) => {
+
+  const userTrials = (await r
```

Comment:
-1 Let's not do a second roundtrip but include this in the `userTiers` query

Reply:
Fixed